html-aam/index.html

```diff
@@ -9365,6 +9365,132 @@ <h4 id="att-controls">`controls`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="att-command">`command`</h4>
+        <table aria-labelledby="att-command">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`command`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td><a data-cite="html/popover.html#attr-command">`button`</a>;</td>
+            </tr>
+            <tr>
+              <th>[[WAI-ARIA-1.2]]</th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>Comments</th>
+              <td>The value of this attribute can impact the mappings of its related <a href="#att-commandfor">`commandfor`</a> attribute.</td>
+            </tr>
+          </tbody>
+        </table>
+        <h4 id="att-commandfor">`commandfor`</h4>
+        <table aria-labelledby="att-commandfor">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`commandfor`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td><a data-cite="html/form-elements.html#attr-commandfor">`button`</a>;</td>
```

Comment:
dupe comment: 
was the support for this removed from "input" buttons ?

the html PR still mentions command / commandfor being allowed on input and button in the indices.html

Reply:
Yes, that sounds like an oversight in the spec. There was a decision to only support the attributes on the button element.